node/assembler/stub_consenter_test.go

```diff
@@ -79,15 +74,15 @@ func NewStubConsenter(t *testing.T, partyID types.PartyID, ca tlsgen.CA) *stubCo
 }
 
 func (sc *stubConsenter) Stop() {
+	sc.server.Stop()
+}
+
+func (sc *stubConsenter) Shutdown() {
```

Comment:
just make sure you cannot restart a stub that was "Shutdown", rather then just "Stopped". 
In Next PR